File: Source/Fantasy.Jobs/Resources/RevokeArgs.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Fantasy.Jobs.Resources
{
    public class RevokeArgs : EventArgs
    {
        internal RevokeArgs ()
	    {
            this.SuspendEngine = true;
	    }

        public bool SuspendEngine { get; set; }
    }
}

File: branches/Stheno/AutoConverted/Fantasy.Jobs/Resources/RevokeArgs.java
package Fantasy.Jobs.Resources;

public class RevokeArgs extends EventArgs
{
	public RevokeArgs()
	{
		this.setSuspendEngine(true);
	}

	private boolean privateSuspendEngine;
	public final boolean getSuspendEngine()
	{
		return privateSuspendEngine;
	}
	public final void setSuspendEngine(boolean value)
	{
		privateSuspendEngine = value;
	}
}